t/01-pdk/08-response/02-get_header.t

```diff
@@ -116,7 +116,7 @@ X-Missing: nil
 
 
 
-=== TEST 4: response.get_header() returns nil when response header does not fit in default max_headers
+=== TEST 4: response.get_header() returns existing header will not limited by get_headers() default max_headers configuration
```

Comment:
Is this a new feature?

Reply:
It's not a new feature, just a refine, since we use `ngx.header[name]` to replace `_RESPONSE.get_headers()[name]` in the `get_header` implementation, the max headers limitation brought by `get_headers()` will not exist anymore.

Reply:
I thinks it's not needed to consider this a new feature, because 
1. the limitation is due to our internal implementation.
2. the limitation is not documented which means users aren't even aware of this limitation.
that means we have no interface changes and no documentation changes from user's perspective.